### ISSUE TITLE
The normalization in `get_features` makes features larger.

### ISSUE THREAD

**Issue by huanglianghua** [visionml/pytracking#12]
Hi, thanks for making the the amazon work open source.

In the following code of [featurebase.py](https://github.com/visionml/pytracking/blob/master/pytracking/features/featurebase.py#L107):

```
# Normalize
if self.normalize_power is not None:
  for feat in feat_list:
    feat /= (torch.sum(feat.abs().view(feat.shape[0],1,1,-1)**self.normalize_power, dim=3, keepdim=True) /
      (feat.shape[1]*feat.shape[2]*feat.shape[3]) + 1e-10)**(1/self.normalize_power)
```

Before normalization, in my run, the `feat.norm()` was `379.7865`, while after the normalization `feat.norm()` was `1381.1995`.

I found that the `(feat.shape[1]*feat.shape[2]*feat.shape[3])` in denominator actually enlarges the `feat` by `(feat.shape[1]*feat.shape[2]*feat.shape[3]) ** (1 / self.normalize_power)` times.

So, is this intended or a mistake? Thanks.

**Comment by martin-danelljan** [comment]
Hi! No this is not a mistake. We want to normalize the power of the features, i.e. the energy per bin. Hence, if one changes the size of the feature map, the magnitude of the feature activations will stay roughly the same.
